### ISSUE TITLE
Emoji as avatar is ignored in integration

### ISSUE THREAD

**Issue by xi67** [RocketChat/Rocket.Chat#29389]
### Description:

Emoji ignored in integration.
The field and in script

### Steps to reproduce:

1. Create a simple integration : 
```
class Script {
  /**
   * @params {object} request
   */
  process_incoming_request({ request }) {
    return {
      content:{
        alias: 'Test',
        emoji: ':grinning:',
        text: 'test emoji'
      }
    };
  }
}
```

2. Call it

### Expected behavior:

The emoji is used as avatar.

### Actual behavior:

The avatar of the user used to post is used.

### Server Setup Information:

- Version of Rocket.Chat Server: 6.1.5
- Operating System: Debian
- Deployment Method: docker
- Number of Running Instances: 1
- DB Replicaset Oplog: wiredTiger
- NodeJS Version: 14.21.2
- MongoDB Version: 5.0.14

### Client Setup Information

- Desktop App : 3.9.4
- Operating System: Win11

**Comment by xi67** [comment]
The rendering is OK from mibile app but not in web browser and desktop app.

**Comment by hugocostadev** [comment]
Fixed on [this PR](https://github.com/RocketChat/Rocket.Chat/pull/28416) and released on the 6.2.0 version
